googleMapsApiModule.py

```diff
@@ -3,14 +3,18 @@
 import config
 import mysql.connector
 
+gmaps = googlemaps.Client(config.key)
+#this line is repeated twice in the code and needs to be globalised
+
+url={'direction':"https://www.google.com/maps/dir/?api=1&" , 'geocoding':"https://www.google.com/maps/search/?api=1&query="}
```

Comment:
As this a constant, I think it'd be better to have a `constants.py` file that contains `BASE_URL = { ... }`, and then call it here like

```python3
from constants import BASE_URL
BASE_URL['direction']
```

Reply:
Good point, why don't you include them @hritikpapa

Reply:
yes I will include this, and make one new file constants.py for all the constants